README.md

```diff
@@ -43,44 +43,42 @@ This project contains 2 packages:
 [npm-java-parser-image]: https://img.shields.io/npm/v/java-parser.svg?color=blue&label=java-parser&logo=java-parser
 [npm-java-parser-url]: https://www.npmjs.com/package/java-parser
 
-## Status
-
-- Parser package alpha version done, it can parse most of Java code. However, we still need to make some tweaks and improvements.
-- The printer (actually `prettier-java` package) is mostly done, it can output formatted code but needs to be improved on some cases.
-
 ## Install
 
 ### Pre-requirements
 
-Since the plugin is meant to be used with Prettier, you need to install it:
-
-`npm install -g prettier`
-
-or
+- Node version 10+
 
-`yarn global add prettier`
+### Install Prettier and Prettier-Java plugin
 
-### Install plugin
+Since the plugin is meant to be used with Prettier, you need to install it, as well as the prettier-plugin-java package.
 
-`npm install -g prettier-plugin-java`
-
-or
+```bash
+npm install prettier prettier-plugin-java --save-dev
 
-`yarn global add prettier-plugin-java`
+# Or globally
+npm install -g prettier prettier-plugin-java
+```
 
-## Usage
+or with yarn:
 
 ```bash
-prettier --write MyJavaFile.java
+yarn add prettier prettier-plugin-java --dev
+
+# Or globally
+yarn global add prettier prettier-plugin-java
 ```
 
-If the plugin is not automatically loaded:
+### Usage
+
+To reformat all you Java files, run:
 
 ```bash
-# Example where the plugin is locate in node_modules
-prettier --write MyJavaFile.java --plugin=./node_modules/prettier-plugin-java
+prettier --write "**/*.java"
```

Comment:
it will probably work only if you install prettier globally, right ? Otherwise, you need a specific "scripts" part inside `package.json`

Reply:
Should be good now :)